packages/neutrino-preset-web/index.js

```diff
@@ -98,10 +95,6 @@ module.exports = (neutrino, opts = {}) => {
       .chunkFilename('[name].[chunkhash].js')
       .end()
     .resolve
-      .alias
-        // Make sure 2 versions of "core-js" always match in package.json and babel-polyfill/package.json
-        .set('core-js', dirname(require.resolve('core-js')))
```

Comment:
I just spotted `core-js` is now unused after this and can be removed from package.json

Reply:
Good catch.